xdocs/usermanual/jmeter_tutorial.xml

```diff
@@ -110,12 +110,27 @@ The folders inside of <code>src</code>
 </p>
 
 <dl>
+<!--
+  <dt><code>bom</code></dt><dd></dd>
+-->
+  <dt><code>bshclient</code></dt><dd>code for the BeanShell based client</dd>
   <dt><code>components</code></dt><dd>contains non-protocol-specific components like visualizers, assertions, etc.</dd>
+<!--
+  <dt><code>config</code></dt><dd>XXX</dd>
+-->
   <dt><code>core</code></dt><dd>the core code of JMeter including all core interfaces and abstract classes.</dd>
+  <dt><code>dist</code></dt><dd>build artifacts will be copied here</dd>
```

Comment:
Technically speaking, `dist` is for a builds script that creates a distribution. So it is the place to look for when you want to update the contents of the resulting source/binary archive.